### PR TITLE
Feature/landing test

### LINKED ISSUE #23: Component: Landing Page
- [x] Layout
- [x] Styling
- [x] Testing
- [x] Breakpoints
- [ ] Clean and push

### PR DESCRIPTION
**Pull request by JamieCaudill**
What I did:

- Added breakpoints for sm, med, lg, and xl screens
- Implemented responsive web design
- Tested Landing, About, and Products components with Cypress
- Applied NavHashLink to Landing button
- Cleaned up code before push
- Closes #23 

Did this break anything?
- [x] No
- [ ] Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

Checklist:
- [x] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [x] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

What's next?
NavBar

### CODE REVIEW

**Review by alivaditis — APPROVED** [review]
Beautiful work Jamie!
One note: It is not immediately clear how a user can scroll through the products on a bigger screen, I could only scroll with keyboard on a bigger screen which took me a while to figure out

File: cypress/e2e/home_spec.cy.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/* eslint-disable no-undef */
+describe('home page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql', {
+      statusCode: 201,
+      fixture: 'items.json'
+    }).as('getItems')
+    cy.wait('@getItems')
```
Comment (taranveersingh93) [inline cypress/e2e/home_spec.cy.js:11]: Nice work waiting for this in beforeEach

File: cypress/e2e/home_spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    cy.get(':nth-child(5) > .card > .card__info > .card__name').should('contain', 'Basket')
+    cy.get(':nth-child(5) > .card > .card__info > .card__price').should('contain', '$30')
+  });
+
+  it('should display an about us section with a header, image, and text', () => {
+    cy.get('.about__header').should('contain', 'About Us')
+    cy.get('.about__header').should('have.css', 'background-image', 'url("http://localhost:3000/static/media/about-bg.f697999f8a20621f9a79.jpg")')
+    cy.get('.about__description-text').should('contain', 'EverUse aims to partner with gyms, retailers, and manufacturers throughout the climbing industry to collect and upcycle used climbing rope into high-quality products. We aim to reduce waste, extend the lifespan of climbing rope, and minimize the environmental footprint of the climbing industry. By offering sustainable products, we strive to become the go-to brand for climbers who are passionate about both their sport and the planet. We aim to form partnerships and provide funding to organizations that are dedicated to maintaining access, restoration, and safety in climbing.')
+  })
+})
```
Comment (taranveersingh93) [inline cypress/e2e/home_spec.cy.js:38]: Could we add a test of clicking on the explore button and checking the url having 'products'?

**Review by taranveersingh93 — APPROVED** [review]
Looks great Jamie. Thanks for dividing this up in sensible it blocks. Would love to add nav arrows for wider screens. Non blocker so approving

File: cypress/e2e/home_spec.cy.js (hunk 2 above)
Comment (JamieCaudill) [reply cypress/e2e/home_spec.cy.js:38]: - [x]

File: cypress/e2e/home_spec.cy.js (hunk 1 above)
Comment (JamieCaudill) [reply cypress/e2e/home_spec.cy.js:11]: Thank you

**Pull request merged**